tools/orca/src/models/markov.rs

```diff
@@ -0,0 +1,143 @@
+use std::{slice::Iter, time::Duration};
```

Comment:
Lets remove/disable this module for now.

Reply:
sure :)